chromium_src/chrome/browser/extensions/extension_tab_util.cc

```diff
@@ -162,7 +162,7 @@ std::unique_ptr<api::tabs::Tab> ExtensionTabUtil::CreateTabObject(
   tab_object->window_id = GetWindowIdOfTab(contents);
   tab_object->status.reset(new std::string(GetTabStatusText(is_loading)));
   tab_object->active = tab_helper->is_active();
-  tab_object->selected = tab_helper->is_active();
+  tab_object->selected = tab_object->active;
```

Comment:
it's the same thing, it just avoids the extra member function call. 
See the line above
`   tab_object->active = tab_helper->is_active();`